virt/install/installing-virt-cli.adoc

```diff
@@ -16,3 +16,12 @@ using the command line to apply manifests to your cluster.
 include::modules/virt-subscribing-cli.adoc[leveloffset=+1]
 
 include::modules/virt-deploying-operator-cli.adoc[leveloffset=+1]
+
+== Next steps
+
+You might want to additionally configure the following components:
+
+* The _KubeMacPool_ component provides a MAC address pool service for virtual machine NICs in designated namespaces. xref:../../virt/virtual_machines/vm_networking/virt-using-mac-address-pool-for-vms.adoc#virt-about-kubemacpool_virt-using-mac-address-pool-for-vms[Enable a MAC address pool in a namespace] by applying the KubeMacPool label to that namespace.
+
+* The xref:../../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-about-hostpath-provisioner_virt-configuring-local-storage-for-vms[hostpath provisioner] is a local storage provisioner designed for {VirtProductName}. If you want to configure local storage for virtual machines, you must enable the hostpath provisioner first.
```

Comment:
forgive my ignorance, but is `{VirtProductName}` meant to be a "drop your virt product name here" space? If so, I would suggest using `<VirtProductName>`. donno, looks more suitable. what do you think?

Reply:
ok sorry only now I found the links to the live links. please ignore this comment.